contributing/great-first-issues-template.md

```diff
@@ -0,0 +1,64 @@
+<!--
+Copy this template into new Great First Issues and edit as needed.
+-->
+<!--
+[Optional] Include this section if you think the issue is perfect for people
+who have never contributed to open source projects before.
+-->
+### First Timers Only
+We know figuring out the process for contributing to an open source project can be intimidating, so we created this issue as a way for you to learn the ropes.  (If you feel comfortable contributing to open source projects, please leave this issue for someone else.)
+
+<!--
+List knowledge the contributor should have before working on this issue.  You can mention that some of these are nice-to-have if you think the knowledge helps but that the issue is still relatively straightforward without that knowledge.
+-->
+### What you will need to know
```

Comment:
I feel 'what you will ned to know' , 'background', 'motivation' and 'the bug' might have overlapping context. Personally I will feel overwhelmed to see so much text to go through and I will probably not read it :) . Maybe we can group some of these and keep it shorter and to the point.